.github/workflows/Windows-CI.yml

```diff
@@ -22,7 +22,7 @@ jobs:
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
 
       - name: Install CMake
-        uses: lukka/get-cmake@5f6e04f5267c8133f1273bf2103583fc72c46b17 #v3.31.5
+        uses: lukka/get-cmake@56d043d188c3612951d8755da8f4b709ec951ad6 #v3.31.6
         with:
           cmakeVersion: 3.31.4
```

Comment:
@BenjamenMeyer The `cmakeVersion` on this line is now two patch versions behind. This needs to be updated manually in every one of these PRs, to keep it in sync with the version on the `uses:` line.